docs/features.md

```diff
@@ -1,16 +1,21 @@
-# Language
-
-* Ceremony master - TBD
-
 # Mainnet Feature Requirements
 
+## Nodes
+### As a Node Operator, I want to install software from binary artifacts or a Docker image
+### As a Node Operator, I want to run software on Linux, MacOS, and in Docker
+### As a dApp Developer, I want to interface with the Rholang interpreter and evaluate smart contracts independently from the blockchain
+### As a Node Operator, I want to have a default configuration and the ability to customize the configuration on the command line
+### As a Node Operator, I want to monitor the performance, resource consumption, and status of my node
+### As a Node Operator, I do not want to expose my private wallet key on the command line
+## Peer to Peer Network
+### As a Node operator, I want to be able to bootstrap to the network by connecting to any known node
+### As a Node operator, once connected via a bootstrap node, I want to discover and connect to peers
+### As a Node operator, I want to know how many peers I am connected to
 ## Network Launch
+### As a Coop SRE I want to launch a network
+#### A succesful genesis ceremony 
```

Comment:
successful

Reply:
Highly recommend [Grammarly](https://chrome.google.com/webstore/detail/grammarly-for-chrome/kbfnbcaeplbcioakkpcpgfkobkghlhen?hl=en).

Reply:
no emacs integration, I've asked ;)